### PR TITLE
Fix new golint warning

### PR DESCRIPTION
**Pull request by stgraber**
Signed-off-by: Stéphane Graber <stgraber@ubuntu.com>

### CODE REVIEW

File: shared/api/response.go [hunk 1]
```diff
@@ -65,11 +65,7 @@ func (r *Response) MetadataAsStringSlice() ([]string, error) {
 
 // MetadataAsStruct parses the Response metadata into a provided struct
 func (r *Response) MetadataAsStruct(target interface{}) error {
-	if err := json.Unmarshal(r.Metadata, &target); err != nil {
-		return err
-	}
-
-	return nil
+	return json.Unmarshal(r.Metadata, &target)
```
Comment (brauner) [inline shared/api/response.go]: Uh, nice that they enforce this.

Comment (stgraber) [reply shared/api/response.go]: Yeah, I just wonder how many more they have :)

**Comment by stgraber** [comment]
jenkins: test this please

**Comment by stgraber** [comment]
We'll need to merge this one and then re-try the rest so we can get clean jenkins results on the others.

**Comment by brauner** [comment]
@stgraber, yup, that was my plan. :)

**Pull request merged**